website/search/elastic_search.py

```diff
@@ -250,9 +248,15 @@ def format_results(results):
         if result.get('category') == 'user':
             result['url'] = '/profile/' + result['id']
         elif result.get('category') == 'file':
-            parent_info = load_parent(result.get('parent_id'))
-            result['parent_url'] = parent_info.get('url') if parent_info else None
-            result['parent_title'] = parent_info.get('title') if parent_info else None
```

Comment:
The old URL's that point to the quickfiles (that are already indexed in elasticsearch) look something like this:
http://localhost:5000/pytusk59c6/files/osfstorage/5d3a44b427fb1202a944ab71/.  The URL references the QuickFilesNode.  

If you navigate there post-migration, you get an error about `'QuickFolder' object has no attribute 'is_public'.  My guess is because you've repointed the QuickFilesNode guid to the Quickfolder? As I pointed out in your data migration, I'd try repointing the QuickFilesNode guid to the User.  (but be sure your reverse migration doesn't delete the user guid, as it's doing now!!)